guides/common/modules/proc_enabling-puppet.adoc

```diff
@@ -0,0 +1,54 @@
+:_module-type: PROCEDURE
+
+[id="proc_enabling-puppet_{context}"]
+= Enabling Puppet Integration with {Project}
+
+[role="_abstract"]
+A fresh installation of {Project} does not have any Puppet integration configured by default.
+You need to enable the integration as is appropriate for your situation.
+{Project} can be configured to manage and deploy Puppet Server on {ProjectServer} or on {SmartProxy}.
+Additionally, Puppet Server can be deployed externally to {Project} and integrated for reporting, facts and external node classification (ENC).
+
+.Prerequisites
+
+* You installed and configured {Project}.
+
+.Procedure
+
+. Enable Puppet on {ProjectServer} before you enable it on {SmartProxy}s:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --enable-foreman-plugin-puppet \
+--enable-foreman-cli-puppet \
+--foreman-proxy-puppet true \
+--foreman-proxy-puppetca true \
+--foreman-proxy-content-puppet true \
+--enable-puppet \
+--puppet-server true \
+--puppet-server-foreman-ssl-ca /etc/pki/katello/puppet/puppet_client_ca.crt \
+--puppet-server-foreman-ssl-cert /etc/pki/katello/puppet/puppet_client.crt \
+--puppet-server-foreman-ssl-key /etc/pki/katello/puppet/puppet_client.key
+----
+
+. Enable Puppet on {SmartProxy}s:
```

Comment:
```suggestion
. Enable Puppet on {SmartProxies}:
```

We have a special attribute for that: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L95